prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -149,12 +150,33 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
         txn: DataAccessTransaction? = null
     ): Boolean {
         val ctx = if (txn != null) DSL.using(txn) else create
+        val weekAgo = OffsetDateTime.now().minusDays(7)
```

Comment:
So duplicates are only check against the last 7 days?  Maybe add a comment here how this is used and why.

Reply:
We don't care about collisions from more than a week ago, and for performance / collision reasons we are only looking within the last 7 days. There is a (low) chance of hash collision so we are limiting that with a time backstop.